copier.yml

```diff
@@ -44,14 +44,14 @@ github_username:
     type: str
     placeholder: "@castelao"
     help: What is your GitHub username?
-version:
-    type: str
-    default: "0.1.0"
-    help: What is your package version?
-use_pytest:
-    type: bool
-    default: true
-    help: Do you want to use pytest for testing?
+# version:
+#     type: str
+#     default: "0.1.0"
+#     help: What is your package version?
+# use_pytest:
```

Comment:
Is your idea to force having tests with `pytest`? Mmmm, why not?

Reply:
I don't think it is worth our time to expand the support for other test systems. The question is - Is it possible a case for a small package without tests? That would justify on adding the setup for `pytest` or nothing.

A reminder for ourselves. It is OK in this repository to be opinionated. We want to make it easier for people to follow what we consider good practices. We don't want to inherit more work to satisfy everyones opinions.

Reply:
I'm having a hard time imagining a repository where you go through the effort of setting up all of this infrastructure but not include tests... 

I would think if you want a quick repository without tests, you probably don't need this template. But I'm open to being persuaded otherwise!